NDcPP_v2_2e.adoc

```diff
@@ -1916,6 +1929,35 @@ _If DTLS is selected in FPT_ITT, then certificate validity is tested in accordan
 
 _The client identifier may be in the Subject field or the Subject Alternative Name extension of the certificate. The expected identifier may either be configured, may be compared to the Domain Name, IP address, username, or email address used by the peer, or may be passed to a directory server for comparison._
 
+*FCS_DTLSS_EXT.2.4* The TSF shall [selection:
+
+* _present a DTLS1.0 Certificate Request message containing the following ClientCertificateTypes: [selection:_
+** rsa_sign,
+** ecdsa_sign,
+** ] and no other ClientCertificateTypes;
+* present a [selection: DTLSv1.2, DTLSv1.3] Certificate Request message containing the following algorithms: [selection:
+** rsa_pkcs1 with sha256(0x0401),
+** rsa_pkcs1 with sha384(0x0501),
+** rsa_pkcs1 with sha512(0x0601),
+** ecdsa_secp256r1 with sha256(0x0403),
+** ecdsa_secp384r1 with sha384(0x0503),
+** ecdsa_secp521r1 with sha512(0x0603),
+** rsa_pss_rsae with sha256(0x0804),
+** rsa_pss_rsae with sha384(0x0805),
+** rsa_pss_rsae with sha512(0x0806),
+** rsa_pss_pss with sha256(0x0809),
+** rsa_pss_pss with sha384(0x080a),
+** rsa_pss_pss with sha512(0x080b)
+** ] and no other algorithms;
+
+].
+
+*_Application Note 54_*
+
+_The selected ClientCertificateTypes and algorithms must be consistent with FCS_COP.1/SigGen and FCS_COP.1/Hash.
+The DTLS 1.2 Certificate Request message includes the algorithms as a list of  SignatureAndHashAlgorithms in the supported signature algorithms list. The signature algorithm is also specified in the certificate_types list.
+The DTLS 1.3 Certificate Request message includes the algorithms as a list of  SignatureSchemes in the signature_algorithms extension._
+
```

Comment:
Added the line breaks and un-italicized the text where required.

Reply:
@plughy2 One more observation here, the _ at the end of the first sentence of Application Note 54 is missing. Also, please remove the # '54' in the app note and replace it with '{counter:appnote_count}_'. Thanks!

Reply:
Moved the "_" to the end of the last sentence since the entire app note should be italicized.  Replaced "Application Note 54" with *_Application Note {counter:appnote_count}_*

Reply:
Thank you @plughy2

Reply:
@plughy2 It should be good now. I added the _ at the beginning and end of all three sentences in FCS_DTLSS_EXT.2.4 app note.